DopplerJobsServer/appsettings.prod.json

```diff
@@ -6,8 +6,8 @@
     "HolidayRetryCountLimit": 5
   },
   "DopplerSapConfiguration": {
-    "CurrencyEndpoint": "http://172.24.16.82:65100/Billing/SetCurrencyRate",
-    "BillingEndpoint": "http://172.24.16.82:65100/Billing/CreateBillingRequest"
+    "CurrencyEndpoint": "http://sap/Billing/SetCurrencyRate",
+    "BillingEndpoint": "http://sap/Billing/CreateBillingRequest"
```

Comment:
Esto requiere configuración en `extra_host` en el Swarm, correcto? ese cambio ya está hacho?

Reply:
En el servicio de Sap que esta en el stack de billing tenemos el `depends_on: -traefik` por lo que nos da la posibilidad de utilizar este tipo de url's como por ejemplo en https://github.com/MakingSense/doppler-jobs/blob/master/DopplerJobsServer/appsettings.prod.json#L3 , estoy en lo correcto?

Reply:
You are right! great!